components/dashboard/src/App.tsx

```diff
@@ -37,7 +37,6 @@ const Projects = React.lazy(() => import(/* webpackPrefetch: true */ './projects
 const Project = React.lazy(() => import(/* webpackPrefetch: true */ './projects/Project'));
```

Comment:
**praise**: Thanks for getting in the project removal (https://github.com/gitpod-io/gitpod/issues/4704)! 💯